init/spark/init.sh

```diff
@@ -4,5 +4,8 @@
 #
 mkdir -p /opt/spark/conf
 cp /tmp/spark/spark-defaults.conf /opt/spark/conf
-wget https://repo1.maven.org/maven2/org/apache/iceberg/iceberg-spark-runtime-3.4_2.12/1.3.1/iceberg-spark-runtime-3.4_2.12-1.3.1.jar  -O /opt/spark/jars/iceberg-spark-runtime-3.4_2.12-1.3.1.jar
```

Comment:
1.3.1 doesn't support Iceberg JDBC catalog

Reply:
Maybe we should merge this pull request after we release the 0.6.0.

Reply:
ok, any other comments?